### PR TITLE
Fix inconsistent card-image height

### PR DESCRIPTION
**Pull request by db118**
Address #56 
Signed-off-by: db118 <drrylbalderas@gmail.com>

## ✅️ By submitting this PR, I have verified the following

* [ X] Checked to [see if a similar PR has already been opened](https://github.com/techqueria/website/pulls) 🤔️
* [X ] Reviewed the [contributing guidelines](https://github.com/techqueria/website/blob/master/CONTRIBUTING.md) 🔍️
* [X ] Added my name to the bottom of the list under the **Credits** section in the [README.md](https://github.com/techqueria/website/blob/master/README.md) with a link to my website or GitHub profile 👥️

### CODE REVIEW

File: src/sass/_card.scss [hunk 1]
```diff
@@ -8,6 +8,26 @@
     padding: 1em;
   }
 
+  .card-image {
+    background-size: 100% 100%;
+    width: 100%;
```
Comment (dvidsilva) [inline src/sass/_card.scss]: Logos still kinda look off for me. Have you tried cover or contain with No repeat?

**Comment by db118** [comment]
I looked into the `background-size: cover` and `background-size: contain`. `cover` cuts off portion of the image and `contain` doesn't full extend into the `card-image` div element

**Comment by dvidsilva** [comment]
I think not fully shown is better than distorted. But we can get others to weight in.

File: src/sass/_card.scss [hunk 2]
```diff
@@ -8,6 +8,28 @@
     padding: 1em;
   }
 
+  .card-image {
+    background-size: 100% 100%;
+    background-position: center;
+    background-repeat: no-repeat;
+    width: 100%;
+    @media screen and (max-width: $mobile) {
+      height: 11em;
```
Comment (fvcproductions) [inline src/sass/_card.scss]: we can actually make this height at 10 em for all viewports and it looks good! 😄

File: site/layouts/partials/content/card/card.event.html [hunk 3]
```diff
@@ -1,10 +1,7 @@
 <div class="card card-equal-height">
   <!-- Image -->
   {{ if .Params.image }}
-  <div class="card-image">
-    <figure class="image">
-      <img src="{{ .Params.image }}" alt="{{ .Title }}">
-    </figure>
+<div class="card-image" style="background-image: url({{.Params.image}})">
```
Comment (fvcproductions) [inline site/layouts/partials/content/card/card.event.html]: missing some spacing in the beginning

File: src/sass/_card.scss [hunk 4]
```diff
@@ -8,6 +8,28 @@
     padding: 1em;
   }
 
+  .card-image {
+    background-size: 100% 100%;
+    background-position: center;
+    background-repeat: no-repeat;
```
Comment (fvcproductions) [inline src/sass/_card.scss]: Can we add the following so it's always a nice perspective?

```
  -webkit-background-size: 100%;
  -moz-background-size: 100%;
  -o-background-size: 100%;
  background-size: 100%;
  background-repeat: no-repeat;
  -webkit-background-size: cover;
  -moz-background-size: cover;
  -o-background-size: cover;
  background-size: cover;
  background-position: center center;
  background-attachment: scroll;
```

Comment (db118) [reply src/sass/_card.scss]: I am curious in why  `background-size` is repeated multiple times

Comment (fvcproductions) [reply src/sass/_card.scss]: oops - you're absolutely right!

should be 

```
  background-repeat: no-repeat;
  -webkit-background-size: cover;
  -moz-background-size: cover;
  -o-background-size: cover;
  background-size: cover;
  background-position: center center;
``

Comment (db118) [reply src/sass/_card.scss]: Gotcha 💃

**Comment by fvcproductions** [comment]
@db118 Thank you for your contribution and for addressing our requests! 

We will want to update the images of some of the sponsors so they can be shown nicely. We will merge this for now and address those later though. 😄 

https://deploy-preview-67--techqueria.netlify.com/sponsors/

**Pull request merged**